sources/host-ctr/cmd/host-ctr/main.go

```diff
@@ -642,6 +642,15 @@ func withBootstrap() oci.SpecOpts {
 		// container's capabilities.
 		seccomp.WithDefaultProfile(),
 		oci.WithAllDevicesAllowed,
+		func(_ context.Context, _ oci.Client, _ *containers.Container, s *runtimespec.Spec) error {
+			// Allow bootstrap containers to manage swap
+			s.Linux.Seccomp.Syscalls = append(s.Linux.Seccomp.Syscalls, runtimespec.LinuxSyscall{
+				Names:  []string{"swapon", "swapoff"},
+				Action: runtimespec.ActAllow,
+				Args:   []runtimespec.LinuxSeccompArg{},
+			})
+			return nil
+		},
```

Comment:
Can you please encapsulate this in a separate function like we do for `withRootFsShared`? https://github.com/bottlerocket-os/bottlerocket/blob/1be9ec869cc825a48da0de9e89905a4e3791e123/sources/host-ctr/cmd/host-ctr/main.go#L841-L849

Reply:
Done, including testing that it still works.